File: AkitaModel/BgNhis/Practice.cs
namespace Skyware.Lis.AkitaModel.BgNhis;

/// <summary>
/// Medical practice, registered in Bulgaria.
/// </summary>
public class Practice
{

    /// <summary>
    /// Practice number (РЦЗ/РЗИ код) of the laboratory, mandatory.
    /// </summary>
    public string PracticeNumber { get; set; } = string.Empty;

    /// <summary>
    /// NHIF number of the practice (НЗОК номер), optional.
    /// </summary>
    public string NhifNumber { get; set; }

    /// <summary>
    /// Laboratory name, optional.
    /// </summary>
    public string Name { get; set; }

}

File: AkitaMotelTS/src/BgNhis/Practice.ts
/**
 * Skyware.Lis.AkitaModel.BgNhis
 */

/**
 * Medical practice, registered in Bulgaria.
 */
export class Practice {
  /**
   * Practice number (РЦЗ/РЗИ код) of the laboratory, mandatory.
   */
  practiceNumber: string = '';

  /**
   * NHIF number of the practice (НЗОК номер), optional.
   */
  nhifNumber?: string;

  /**
   * Laboratory name, optional.
   */
  name?: string;
}
